pkg/webhook/server.go

```diff
@@ -132,6 +134,7 @@ func (s *Server) Start(stop <-chan struct{}) error {
 		if _, err := inject.LoggerInto(baseHookLog.WithValues("webhook", hookPath), webhook); err != nil {
 			return err
 		}
+		baseHookLog.Info("loaded webhook in", "webhook path", hookPath)
```

Comment:
IMO we should put it in https://github.com/kubernetes-sigs/controller-runtime/blob/13ee2bc1b81d8c3ef01ea48d28c4028933865e5b/pkg/webhook/server.go#L100
```
baseHookLog.Info("registering webhook", "path", path)
s.webhooks[path] = hook
s.WebhookMux.Handle(path, instrumentedHook(path, hook))
```